l/libsass/stone.yaml

```diff
@@ -0,0 +1,23 @@
+#
+# SPDX-FileCopyrightText: © 2020-2025 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : libsass
+version     : 3.6.6
+release     : 1
+homepage    : https://github.com/sass/libsass
+upstreams   :
+    - https://github.com/sass/libsass/archive/3.6.6.tar.gz : 11f0bb3709a4f20285507419d7618f3877a425c0131ea8df40fe6196129df15d
+summary     : A C/C++ implementation of a Sass compiler
+description : |
+    A C/C++ implementation of a Sass compiler
+license     : MIT
+setup       : |
+    %reconfigure --disable-static
+build       : |
+    %make
+install     : |
+    %make_install
+# hardcoded to gcc-ar
```

Comment:
I looked into this and the issue was actually in slibtool. I added a patch to it to fix it, can you remove this line and add a depends on https://github.com/AerynOS/recipes/pull/709 instead? You'll need to rebuild this package to pick up that it's linking to libc++ and libunwind (I tested it and it builds with updated slibtool).

Reply:
Also if binutils ar is the only thing you need from the gnu stack it's better to use `binary(ar)` as a builddep.